source/includes/fundamentals/code-snippets/auth-credentials-gssapi-subject-key.rst

```diff
@@ -0,0 +1,9 @@
+.. code-block:: java
+
+   LoginContext loginContext = new LoginContext(<LoginModule implementation from JAAS config>);
+   loginContext.login();
+   Subject subject = loginContext.getSubject();
+
+   MongoCredential credential = MongoCredential.createGSSAPICredential(<username>);
+   credential = credential.withMechanismProperty(MongoCredential.JAVA_SUBJECT_KEY, subject);
```

Comment:
Question: Should these be chained? (with a new line for the chained method)
```suggestion
   MongoCredential credential = MongoCredential.createGSSAPICredential(<username>).withMechanismProperty(MongoCredential.JAVA_SUBJECT_KEY, subject);
```

Reply:
I think they could be, but the example in this part of the page that was already there had them broken into two lines, so I tried to replicate the style. I think it is ok to leave it as two lines.